powerhome/playbook
# [Play-528] Typescript Conversion Stat Change

#### Screens

<img width="355" alt="Screen Shot 2023-02-03 at 2 10 17 PM" src="https://user-images.githubusercontent.com/73671109/216687125-486fc57b-796b-4152-8b4f-ce4585a5603f.png">

#### Breaking Changes

NO

#### Runway Ticket URL

[Runway Ticket](https://nitro.powerhrg.com/runway/backlog_items/PLAY-528)

#### How to test this

Check to see if stat change kit still works as expected after typescript conversion.

#### Checklist:
